tools/socks/main.go

```diff
@@ -23,11 +23,11 @@ const (
 	pacfile = `
 function FindProxyForURL(url, host) {
 	if(shExpMatch(host, "{{.HostMatch}}")) {
-		return "SOCKS5 localhost:8000";
+		return "SOCKS5 127.0.0.1:8000";
```

Comment:
Please make these changes in tools (although I don't think it's needed in this case since we only use this internally).